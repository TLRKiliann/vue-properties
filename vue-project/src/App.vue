<script lang="ts">
  import FirstComp from './components/FirstComp.vue'
  import SecondComp from './components/SecondComp.vue'
  import { defineComponent } from 'vue'
  export default defineComponent({
    name: "App",
    components: {
      FirstComp,
      SecondComp
    },
    data() {
      return {
      }
    }
  })
</script>

<template>
  <div class="main--div">
    <FirstComp name="Bruce" />
    <FirstComp name="Diana" />
    <FirstComp name="name" />
    <SecondComp firstName="Esteban" lastName="Catanea" />
  </div>
</template>

<style scoped>
  .main--div {
    width: 100%;
    margin: 0 50%;
  }
</style>
