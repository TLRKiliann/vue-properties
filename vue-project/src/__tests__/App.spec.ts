import { it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import App from '../App.vue'

it('snapshot UI testing', () => {
    const wrapper = mount(App,{});
    expect(wrapper.text()).toMatchSnapshot()
})