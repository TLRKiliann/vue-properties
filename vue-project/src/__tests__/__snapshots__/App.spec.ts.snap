// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot UI testing 1`] = `"Name : BruceName : DianaName : nameFirstname: Esteban Lastname: CataneaFullname: Esteban CataneaList of itemsPC 1900.- Delete an item Phone 400.- Delete an item Laptop 1200.- Delete an item Computed : 3500.-Method : 3500.- Add an item"`;
